src/GrumPHP/Process/ProcessBuilder.php

```diff
@@ -72,4 +80,14 @@ private function getCommandLocation($command)
     {
         return $this->externalCommandLocator->locate($command);
     }
+
+    /**
+     * @param Process $process
+     */
+    private function writeVerboseOutput($process)
+    {
+        if ($this->io->isVeryVerbose()) {
+            $this->io->write(PHP_EOL.'Command: '.$process->getCommandLine(), true);
```

Comment:
Why did you add PHP_EOL? The last parameter of the method is already adding a newline.